### ISSUE TITLE
UI and accessibility

### ISSUE THREAD

**Issue by rtvb** [eclipse/smarthome#1647]
UI's, including icons, should be created with [accessibility](https://en.wikipedia.org/wiki/Accessibility) in mind.

In issue https://github.com/eclipse/smarthome/issues/1614#issuecomment-223986194 @digitaldan refers to using green and red icons being an issue for the color blind among us. I think he has a strong point and created this issue as a placeholder for future discussion.
